lib/Core/Site/QueryType/Content/Relations/AllTagFields.php

```diff
@@ -0,0 +1,93 @@
+<?php
```

Comment:
Added in https://github.com/netgen/ezplatform-site-api/pull/70/commits/e0cc48ff75986bb8d68e505b336344b32f392a11

Reply:
This query type will return all content which contains the corresponding tags, including the current (context) content. Since this represents a relation based on the current content, I think that there's no need to return the current content itself.

For example, when you have an article and you want to fetch related articles based on tags. There's no need to get the current article as a related one as well.

PS. same goes for TagFields query type as well.

Reply:
Added in https://github.com/netgen/ezplatform-site-api/pull/70/commits/11eb4a24d755a6850e244f00c5c9bd0b8f32ea15.

I also added `exclude_context` option to control that behavior, defaulting to `true`.
Not sure if the name is good enough, I'm open for better suggestions.

Will also update `Subtree` with `exclude_root = true` instead of `include_root = false`.